platform/ios/src/MGLLocationManager.m

```diff
@@ -1,9 +1,10 @@
 #import "MGLLocationManager.h"
 #import <UIKit/UIKit.h>
 
-static const NSTimeInterval fiveMinuteTimeInterval = 300.0;
-static const NSTimeInterval fiveSecondTimeInterval = 5.0;
-static const CLLocationDistance regionRadiusLocationDistance = 300.0;
+static const NSTimeInterval MGLLocationManagerFiveMinuteTimeInterval = 300.0;
```

Comment:
While we’re at it, can we give these constants more meaningful names? If not, they should be inlined.